app/views/spree/admin/payments/show.html.haml

```diff
@@ -1,4 +1,5 @@
-= render partial: 'spree/admin/shared/order_tabs', locals: { current: 'Payments' }
+= render :partial => 'spree/admin/shared/order_page_title', locals: { current: 'Payments' }
+= render :partial => 'spree/admin/shared/order_tabs', locals: { current: 'Payments' }
```

Comment:
basically, you have to do this (call both partials) in the other partials that call order_tabs.

Reply:
I had tried that but got duplicate text (e.g. new order New order #xxxxx, etc). Cleaned it all up, put the two renders back in and looks good on my local now.